src/libs/formatRighTextUtil.ts

```diff
@@ -0,0 +1,24 @@
+const { load } = require('cheerio')
+import highlight from 'highlight.js'
+import 'highlight.js/styles/night-owl.css'
+
+export const formatRichText = (richText: string) => {
```

Comment:
I borrowed the almost same code from microcms's template. 
https://github.com/microcmsio/nextjs-simple-blog-template/blob/main/libs/utils.ts

some of the codes were already deprecated, so I updated a little.